control-plane/build-support/functions/10-util.sh

```diff
@@ -108,70 +105,36 @@ function parse_version {
 	#   If the GOTAGS environment variable is present then it is used to determine which
 	#   version file to use for parsing.
 
-	local vfile="${1}/version/version.go"
+	local vfile="${1:-.}/version/VERSION"
 
 	# ensure the version file exists
 	if ! test -f "${vfile}"; then
 		err "Error - File not found: ${vfile}"
 		return 1
 	fi
 
-	local include_release="$2"
-	local use_git_env="$3"
 	local omit_version="$4"
 
 	local git_version=""
 	local git_commit=""
 
-	if test -z "${include_release}"; then
-		include_release=true
-	fi
-
-	if test -z "${use_git_env}"; then
-		use_git_env=true
-	fi
-
-	if is_set "${use_git_env}"; then
-		git_version="${GIT_DESCRIBE}"
-		git_commit="${GIT_COMMIT}"
-	fi
-
 	# Get the main version out of the source file
-	version_main=$(awk '$1 == "Version" && $2 == "=" { gsub(/"/, "", $3); print $3 }' <${vfile})
-	release_main=$(awk '$1 == "VersionPrerelease" && $2 == "=" { gsub(/"/, "", $3); print $3 }' <${vfile})
-
-	# try to determine the version if we have build tags
-	for tag in "$GOTAGS"; do
-		for vfile in $(find "${1}/version" -name "version_*.go" 2>/dev/null | sort); do
-			if grep -q "// +build $tag" "${vfile}"; then
-				version_main=$(awk '$1 == "Version" && $2 == "=" { gsub(/"/, "", $3); print $3 }' <${vfile})
-				release_main=$(awk '$1 == "VersionPrerelease" && $2 == "=" { gsub(/"/, "", $3); print $3 }' <${vfile})
-			fi
-		done
-	done
```

Comment:
This was `consul` copypasta by Mitchell and never used from what I can see (we don't have any files that match those globs)